Wikipedia/Code/EmptyRefreshingViewController.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+protocol EmptyRefreshingViewControllerDelegate: class {
```

Comment:
`class` is supposed to be deprecated in favor of `AnyObject` in future versions of Swift https://forums.swift.org/t/class-only-protocols-class-vs-anyobject/11507

Reply:
@nambatee good to know, done!